Networking/Networking/Remote/WCAnalyticsCustomerRemote.swift

```diff
@@ -15,13 +17,34 @@ public class WCAnalyticsCustomerRemote: Remote {
             return
         }
 
+        enqueueRequest(with: ["search": name], siteID: siteID, completion: completion)
+    }
+
+    /// Loads a paginated list of customers
+    /// 
+    public func loadCustomers(for siteID: Int64,
+                                pageNumber: Int = 1,
+                                pageSize: Int = 25,
+                                completion: @escaping (Result<[WCAnalyticsCustomer], Error>) -> Void) {
+        let parameters = [
+            ParameterKey.page: String(pageNumber),
+            ParameterKey.perPage: String(pageSize),
+            ParameterKey.orderBy: "name",
+            ParameterKey.order: "asc",
```

Comment:
```suggestion
            ParameterKey.order: "desc",
```
The only way I could make this work while testing is switching the order to descending, since that would load the most recent customers, otherwise the response would return empty: 

`customers" [" "]`

I think it could be related to us [not mapping unregistered customers](https://github.com/woocommerce/woocommerce-ios/issues/7951) in the past, but since the [API changes](https://github.com/woocommerce/woocommerce/pull/36216) those are part of the results from wc-analytics now, so in my case the response for the first page was being populated with all non-registered customers?

I'd say that makes sense to change the order to descending, but I'm not sure is what we want to do?